openhands/agenthub/codeact_agent/function_calling.py

```diff
@@ -521,3 +521,11 @@ def get_tools(
     else:
         tools.append(StrReplaceEditorTool)
     return tools
+
+
+def action_to_str(action: Action) -> str:
+    # for example, when the user runs a command from the terminal
+    # we want to show the command in the chat history
+    if isinstance(action, CmdRunAction) and action.source == 'user':
+        return f'COMMAND:\n{action.command}'
```

Comment:
If this `action_to_str` is only used for converting user command to string for visualization, can we do something like:

```
User executed the command: {action.command}
```

And can we add a prefix for the CmdRun observation that results from this? those observation would have `tool_call_metadata` being empty, and not sure if it will cause issue

Reply:
They worked fine with empty, in my tests. We merge the message, it's only one being sent for both of them. 🤔 

Adding that text for the LLM to see.